src/withHelmet.js

```diff
@@ -0,0 +1,14 @@
+/* eslint react/display-name: 0 */
+import React from "react";
+import createElement from "recompose/createElement";
+import createHelper from "recompose/createHelper";
+import Helmet from "./Helmet.js";
+
+const withHelmet = (propsMapper, BaseComponent) => props => (
+  <div>
```

Comment:
Yes, the empty `div` wrapper may cause side effect in some styling case.
But I have not come up with idea to work with `react-side-effect`.
Any suggestion to improve it ?